include/boost/http_proto/message_view_base.hpp

```diff
@@ -91,6 +92,15 @@ class BOOST_SYMBOL_VISIBLE
     {
         return ph_->md;
     }
+
+    /** Return true if the message is using a chunked
+        transfer encoding.
+    */
+    bool
+    chunked() const noexcept
```

Comment:
should this be `is_chunked`? I am not sure.. what do the other members look like?

Reply:
I chose `chunked()` because there's `keep_alive()` instead of `is_keep_alive()`.
```cpp
/** Return true if semantics indicate connection persistence
*/
bool
keep_alive() const noexcept
{
    return ph_->keep_alive();
}
```